### PR TITLE
Xfce raw images

### PR DESCRIPTION
**Pull request by vinzv**
New post about Xfce virtualization images

### CODE REVIEW

File: _posts/2020-11-23-xfce-raw-images.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ You can find more detailed information on how to use these images in the dedicat
 
 ## Contributing
 
-Like any other team in openSUSE the Xfce team is always happy to welcome people interested in development, packaging, testing and reporting bugs. Find out more about our work [here](https://en.opensuse.org/Portal:Xfce) and say hello in the chat if you like! [7]
+Like any other team in openSUSE the Xfce team is always happy to welcome people interested in development, packaging, testing and reporting bugs. Find out more about our work [here](https://en.opensuse.org/Portal:Xfce) and say hello in the chat if you like!
 
 
-[1] RAT stands for Released Ahead of Time. Courtesy of this word play is LCP.
+<sup>[1] RAT stands for Released Ahead of Time. Courtesy of this word play is LCP.</sup>
```
Comment (hellcp) [inline _posts/2020-11-23-xfce-raw-images.md:53]: https://kramdown.gettalong.org/quickref.html#footnotes

File: _posts/2020-11-23-xfce-raw-images.md [hunk 2]
```diff
@@ -50,4 +50,4 @@ You can find more detailed information on how to use these images in the dedicat
 Like any other team in openSUSE the Xfce team is always happy to welcome people interested in development, packaging, testing and reporting bugs. Find out more about our work [here](https://en.opensuse.org/Portal:Xfce) and say hello in the chat if you like!
 
 
-<sup>[1] RAT stands for Released Ahead of Time. Courtesy of this word play is LCP.</sup>
+<sup>[^1] RAT stands for Released Ahead of Time. Courtesy of this word play is LCP.</sup>
```
Comment (hellcp) [inline _posts/2020-11-23-xfce-raw-images.md:53]: `[^1]:`, not `[^1]`

**Comment by vinzv** [comment]
Looks like the footnote thing is not working... :shrug: @hellcp

**Comment by m4u9** [comment]
LGTM

**Pull request merged**